src/cozmo/util.py

```diff
@@ -134,6 +134,12 @@ def degrees(self):
         '''float: The angle in degrees.'''
         return self._radians / math.pi * 180
 
+    @property
+    def abs_val(self):
```

Comment:
can we rename this to abs_value. Also this really needs a docstring as it's part of the public SDK API